### PR TITLE
Add pytorch-tpu/examples to the docker image.

### PR DESCRIPTION
**Pull request by taylanbil**
this will enable testing the examples, which include fairseq..

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -15,7 +15,8 @@ RUN git clone https://github.com/pytorch/pytorch
 COPY . /pytorch/xla
 RUN if [ "${cloud_build}" = true ]; then github_branch="${release_version}" && \
   if [ "${release_version}" = "nightly" ]; then github_branch="master"; fi && \
-  cd /pytorch && rm -rf xla && git clone -b "${github_branch}" --recursive https://github.com/pytorch/xla; fi
+  cd /pytorch && rm -rf xla && git clone -b "${github_branch}" --recursive https://github.com/pytorch/xla && \
+  cd / && git clone -b "${github_branch}" --recursive https://github.com/pytorch-tpu/examples tpu-examples; fi
```
Comment (dlibenzi) [inline docker/Dockerfile]: Can we maybe format this such that every command is on one line?
```
  cmd1 && \
    cmd2 && \
    cmd3 && \
    ...
```

Comment (taylanbil) [reply docker/Dockerfile]: how does it look now?

**Comment by taylanbil** [comment]
tested using `docker build -t torch-xla -f docker/Dockerfile .` and then `docker run -it torch-xla:latest`, the folder /tpu-examples is present (after setting cloud-build to true in Dockerfile)

**Comment by taylanbil** [comment]
failure is unrelated, merging.
```
[  FAILED  ] 1 test, listed below:
[  FAILED  ] AtenXlaTensorTest.TestPinverse
```

**Pull request merged**